optaweb-vehicle-routing-backend/config/checkstyle/Checker.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.2//EN"
+        "https://checkstyle.org/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+    <!-- Checks for whitespace. -->
```

Comment:
Can you please use 2-space indentation for this file? We use it for all XML files.